src/components/ContentList/ContentList.scss

```diff
@@ -16,6 +16,11 @@ $marginIconSizeL: 1px;
             @include text-size(header-1);
 
             margin: 0 0 $indentXXXS;
+
+            .yfm {
```

Comment:
you have to add it in size_s too

Reply:
added

Reply:
Can we set this styles in one place. I see copy-paste in size l and s